internal/backend/db/dbgorm/ownership.go

```diff
@@ -120,40 +121,52 @@ func (gdb *gormdb) createEntityWithOwnership(
 	}
 
 	return gdb.transaction(ctx, func(tx *tx) error {
-		if err := tx.isUserEntity(user, idsToVerifyOwnership...); err != nil {
+		if err := tx.isUserEntity(uid, idsToVerifyOwnership...); err != nil {
 			return err
 		}
-		if err := create(tx.db, user); err != nil { // create
+		if err := create(tx.db, uid); err != nil { // create
 			return err
 		}
-		return saveOwnershipForEntities(tx.db, user, ownerships...) // ensure user and add ownerships
+		return saveOwnershipForEntities(tx.db, uid, ownerships...) // ensure user and add ownerships
 	})
 }
 
-func isUserEntity(db *gorm.DB, uid string, ids ...sdktypes.UUID) error {
+func getOwnerships(db *gorm.DB, ids ...sdktypes.UUID) ([]scheme.Ownership, error) {
```

Comment:
maybe getEntitiesOwnership is better name
since you can query by entity_id or by user_id

Reply:
getOwnerships is good enough IMO since we are passing ids of the entities to get ownerships for

Reply:
getOwnerships not getting any user_id, just a list of entities ids

Reply:
you are passing ids as a general
userIDs and entityIDs in this case are the same

Reply:
but the query is entity_id =?

Reply:
I could rename ids to entitiesIDs if this makes it better

Reply:
renamed  to `getOwnershipForEntities`